examples/bin/load_test

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+require_relative '../init'
+
+Dir[File.expand_path('../workflows/*.rb', __dir__)].each { |f| require f }
+
+workflow_class_name, *args = ARGV
+workflow_class = Object.const_get(workflow_class_name)
+
+
+# Convert integer strings to integers
+input = args.map { |arg| Integer(arg) rescue arg }
+
+while true
+  sleep(1)
+  workflow_id = SecureRandom.uuid
+  run_id = Cadence.start_workflow(workflow_class, *input, options: { workflow_id: workflow_id })
+  Cadence.logger.info "Started workflow: #{run_id} / #{workflow_id}"
+end
+
```

Comment:
^ This file doesn't seem to be related to this PR

Reply:
Yeah not sure how this got in here... I'll remove it